contracts/Comptroller/ComptrollerStorage.sol

```diff
@@ -264,3 +265,8 @@ contract ComptrollerV14Storage is ComptrollerV13Storage {
     /// @notice Prime token address
     IPrime public prime;
 }
+
+contract ComptrollerV15Storage is ComptrollerV14Storage {
```

Comment:
Should we change the references to `ComptrollerV14Storage` also in these files?
* `contracts/Comptroller/Diamond/interfaces/IRewardFacet.sol`
* `contracts/Comptroller/Diamond/interfaces/ISetterFacet.sol`

Would it make sense to change them to `ComptrollerV9Storage`, where the `Action` enum was defined?

Reply:
Wdyt? [9fdc436](https://github.com/VenusProtocol/venus-protocol/pull/380/commits/9fdc436c625f68306eacc679ed8bb448958a3246)